README.md

```diff
@@ -1,7 +1,7 @@
 OpenStudio
 ==========
 
-OpenStudio is a cross-platform (Windows, Mac, and Linux) collection of software tools to support whole building energy modeling using [EnergyPlus](https://github.com/NREL/EnergyPlus) and advanced daylight analysis using [Radiance](https://github.com/NREL/Radiance/).  OpenStudio is an open source project to facilitate community development, extension, and private sector adoption.
+OpenStudio is a cross-platform (Windows, Mac, and Linux) collection of software tools to support whole building energy modeling using [EnergyPlus](https://github.com/NREL/EnergyPlus) and advanced daylight analysis using [Radiance](https://github.com/LBNL-ETA/Radiance).  OpenStudio is an open source project to facilitate community development, extension, and private sector adoption.
```

Comment:
Your contribution is appreciated. But we actually use this release: https://github.com/NREL/Radiance/releases/tag/5.0.a.12

As can be seen here: https://github.com/NREL/OpenStudio/blob/023c513504a23125d2c2232f693d8ab6e6bad1b8/CMakeLists.txt#L208

So I'd say we shouldn't update the link.

Reply:
You'll see that on the LBNL repo, which is up to date, the oldest release is [Radiance 5.3a (2020-07-11)](https://github.com/LBNL-ETA/Radiance/releases/tag/5.3a.4b97769a)